src/components/Menu.stories.tsx

```diff
@@ -1,44 +1,97 @@
-import type {Meta, StoryObj} from "@storybook/react";
-import Menu from "./Menu";
+import type { Meta, StoryObj } from "@storybook/react";
+import { Menu, Message} from "./Menu";
+import { AuthProvider } from "../contexts/AuthContext";
+
+const mockLocations = [
+    { id: "1", name: "Pine Valley Golf Club" },
+    { id: "2", name: "Augusta National Golf Club" },
+    { id: "3", name: "Pebble Beach Golf Links" },
+];
+
+const mockMessagesData: Message[] = [
+    {
+        clientName: "Bob Buddy Boy",
+        clientImage: 'https://example.com/image1.jpg',
+        unreadCount: 0,
+        lastMessageAt: Date.now() - 68000,
+        lastMessage: "Short message",
+        locationId: "1",
+    },
+    {
+        clientName: "Bobby Bot",
+        clientImage: 'https://example.com/image1.jpg',
+        unreadCount: 70,
+        lastMessageAt: Date.now() - 95000,
+        lastMessage: "Short message",
+        locationId: "1",
+    },
+    {
+        clientName: "Carson Carl Jr.",
+        clientImage: 'https://example.com/image1.jpg',
+        unreadCount: 2,
+        lastMessageAt: Date.now() - 6800000,
+        lastMessage: "This is a long message, where the sentence is very redundant and long because it is long",
+        locationId: "2"
+    },
+    {
+        clientName: "Kaytlyn Lyn Lynda",
+        clientImage: 'https://example.com/image1.jpg',
+        unreadCount: 50,
+        lastMessageAt: Date.now() - 2400000,
+        lastMessage: "this is a long message, where the sentence is very redundant and long because it is long",
+        locationId: "3"
+    },
+];
+
+function getMessagesForSelected(selected: string | null) {
+    if (!selected) return [];
+    return mockMessagesData.filter(msg => msg.locationId === selected);
+  }
+  
 
 const meta: Meta<typeof Menu> = {
     title: "Components/Menu",
     component: Menu,
-};
+    decorators: [
+        (Story) =>(
+            <AuthProvider>
+                <Story />
+            </AuthProvider>
+        )
+    ]
+  };
 
 export default meta;
+
 type Story = StoryObj<typeof Menu>;
 
 export const Default: Story = {
     args: {
-        messagesData: [],
-    }
+      selected: null,
+      locations: [],
+    },
+};
+
+export const PineValleyGolfClub: Story = {
```

Comment:
Looks like the placeholder is now broken since we're trying to pull in an image that doesn't exist: 
<img width="1040" alt="image" src="https://github.com/user-attachments/assets/443b46d1-f8b3-4930-8636-abc601218a60" />
We can use something like [a placeholder image](https://placehold.co/) or put in the Base64 icons to fix this up. 
However, since this is just the storybook and this PR has been open a while we can go ahead and merge this! @ChristianSTo can you create a small jira bug to fix this storybook issue?